### DISCUSSION TITLE
Unable to download from Vidsrc

### DISCUSSION THREAD

**Discussion by imperfectax** [aclap-dev/video-downloadhelper#1447] [Help and Q&A]
### Your question

Hello, I have been using VDH for Vidsrc ever since I found it. Whenever I try to download something this few days from vidsrc it says download failed.  Here's an example link. https://vidsrc.net/embed/tt0073195  I didn't see any recent threads about vidsrc so couldn't tell if it's a known issue or not. Having same issue on both the stable and beta release. Tried it on Edge as well with the same issue.  Any help at all would be greatly appreciated!

Thanks!

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"abae8c3d83914e3b"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:132.0) Gecko/20100101 Firefox/132.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "C:\\Users\\ebbmi\\dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\ebbmi\\dwhelper\\AfterMASH"
        }
    },
    "concurrent_downloads_max": 6,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "use_wide_ui": true,
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 13,
    "successfull_dl": 1,
    "never_show_successfull_dl_message": true,
    "record_download_history": false
}

**Comment by Mattmjs** [comment]
#1383 if you searched the forum before posting.
